styles/App.scss

```diff
@@ -1,3 +1,30 @@
 body {
     font-family: 'Montserrat', sans-serif;
 }
+
+
+.movie-tiles {
```

Comment:
We should put styles per component
So it should be tileComponent.scss in folder when TileComponent.js is
Also any color, background-color etc. should be taken from variables

Reply:
I've made a new file for this in the same folder, now styles are in good place